### PR TITLE
Issue #3209094 by Ressinel: Private group member doesn’t see group content on the search page (revert)

### PR DESCRIPTION
**Pull request by Ressinel**
## Problem
Follow up: https://github.com/goalgorilla/open_social/pull/2295

Change in node grand access checks is really heavy and causes performance issues.

## Solution
Temporarily revert changes until finding a solution that improves performance for it.

## Issue tracker
- https://getopensocial.atlassian.net/browse/YANG-6053
- https://www.drupal.org/project/social/issues/3209094

## How to test
- [ ] Pending better site performance

## Screenshots
N/A

## Release notes
N/A

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2438-0pcd7s7mn7xlfriat4babdbezxrbyogn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60fe74da80afb01c914a2ffb

File: modules/custom/entity_access_by_field/entity_access_by_field.post_update.php [hunk 1]
```diff
@@ -8,6 +8,6 @@
 /**
  * Rebuild node access.
  */
-function entity_access_by_field_post_update_10101_rebuild_node_access() {
+function entity_access_by_field_post_update_10102_rebuild_node_access() {
```
Comment (navneet0693) [inline modules/custom/entity_access_by_field/entity_access_by_field.post_update.php:11]: @Ressinel Please don't rename the existing one, instead, simply, add the comment link to this PR in the preview hook and create a new hook for rebuilding access.

Comment (Ressinel) [reply modules/custom/entity_access_by_field/entity_access_by_field.post_update.php:19]: Check it, please.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2438-0pcd7s7mn7xlfriat4babdbezxrbyogn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60fe74da80afb01c914a2ffb

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2438-0pcd7s7mn7xlfriat4babdbezxrbyogn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60fe74da80afb01c914a2ffb

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2438-0pcd7s7mn7xlfriat4babdbezxrbyogn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60fe74da80afb01c914a2ffb

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2438-0pcd7s7mn7xlfriat4babdbezxrbyogn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60fe74da80afb01c914a2ffb

**Review by navneet0693 — APPROVED** [review]

**Comment by navneet0693** [comment]
🍒 - picked to 10.2.x and 10.1.x

**Pull request merged**